spec.md

```diff
@@ -365,15 +368,68 @@ allowing a long string to be split across multiple lines of the source file.
 def"			# "abcdef"
 ```
 
-An *octal escape* encodes a single byte using its octal value.
+An *octal escape* encodes a single string element using its octal value.
 It consists of a backslash followed by one, two, or three octal digits [0-7].
```

Comment:
> followed by one, two, or three octal digits

This is precedent for the problem we discussed earlier today about `\XFF` vs `\XFFFF`. We could just consume four hex digits if possible, then three if possible, and finally two, erroring out if that's not possible. Then you don't need to know whether k=8 or k=16 to parse `\X` escapes, and you aren't forced to write `\X00FF` instead of `\XFF`.

Still doesn't change the other problems we discussed about the ambiguity of the denotation, I think, so I guess it's moot.

Reply:
During discussions, we have decided to abandon `\X` element escapes, since they are inherently non-portable, so this is indeed moot.